src/functions-and-arrays.js

```diff
@@ -29,9 +105,36 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(array) {
+  let uniqueArray = [];
+  if (array.length === 0) {
+    return null;
+  }
+  uniqueArray = Array.from(new Set(array));
+  return uniqueArray;
+
```

Comment:
Hola @santiagoacasa , que querias hacer aca?
Recibis un array, lo convertis en un objeto y luego de vuelta a un array y devolves ese array

Reply:
https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Set
https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Array/from

Investigando un poco vi que con Set al pasarle un objeto iterable crea un objeto con los valores encontrados. Pero como cada valor tiene que ser único en el set elimina los repetidos. 
Y con Array.from creo un array con esos datos.

Reply:
Si, efectivamente, muy original, no me di cuenta!
Te felicito por la creatividad.

Un objeto se puede loopear con un for in loop, volverlo iterable significa agregarle una propiedad al objeto bajo el nombre Simbol.iterator y esta convertirla en una function generator. 
Esto te permite hacer esto ---> [...obj] lo cual te permite hacer lo que bien hiciste ---> Array.from.
Al volverlo iterable por keys suprimiste las repetidas, genial.

Te invito a que, despues del curso, leas sobre las function generators y la propiedad Simbol.iterator